### DISCUSSION TITLE
v1.3.1.1

### DISCUSSION THREAD

**Discussion by xmine64** [MicaForEveryone/MicaForEveryone#306] [Announcements]
* **NOTE**: The packaged version has an updated certificate, you would need to reinstall the certificate to be able to upgrade. For certificate installation instructions see [here](https://github.com/MicaForEveryone/MicaForEveryone/wiki/Installation#msix-file)
____

## What's Changed
* Resorted Translator list by @toineenzo in https://github.com/MicaForEveryone/MicaForEveryone/pull/277
* Stability and Performance improvements merged from dev branch (#271)
* Experimental support for auto-reload after changing language

### Dependencies
* build(deps): bump CommunityToolkit.Mvvm from 8.0.0 to 8.1.0 by @dependabot in https://github.com/MicaForEveryone/MicaForEveryone/pull/227
* build(deps): bump XclParser from 1.0.7 to 1.0.8 by @dependabot in https://github.com/MicaForEveryone/MicaForEveryone/pull/273
* build(deps): bump vedantmgoyal2009/winget-releaser from 1 to 2 by @dependabot in https://github.com/MicaForEveryone/MicaForEveryone/pull/252
* build(deps): bump CommunityToolkit.Mvvm from 8.1.0 to 8.2.0 by @dependabot in https://github.com/MicaForEveryone/MicaForEveryone/pull/287

### Translations
* Translation Traditionnal Chinese by @MW1Z in https://github.com/MicaForEveryone/MicaForEveryone/pull/226
* Ukrainian localization by @Phyan in https://github.com/MicaForEveryone/MicaForEveryone/pull/250
* Fix Traditional Chinese by @flandretw in https://github.com/MicaForEveryone/MicaForEveryone/pull/261
* Indonesian translation branch by @seanardhana in https://github.com/MicaForEveryone/MicaForEveryone/pull/264
* feat(UI/Strings/tr): Add Turkish translation by @baglayan in https://github.com/MicaForEveryone/MicaForEveryone/pull/269
* Added Hungarian translation by @Zan1456 in https://github.com/MicaForEveryone/MicaForEveryone/pull/270
* Fixed typos in Ukrainian translation by @lowl1f3 in https://github.com/MicaForEveryone/MicaForEveryone/pull/282
* Correcting and make word easier to understand (ID) by @GID0317 in https://github.com/MicaForEveryone/MicaForEveryone/pull/283
* fix(UI/Strings/zh-Hant): update translations by @jay900604 in https://github.com/MicaForEveryone/MicaForEveryone/pull/284
* Serbian (Latin and Cyrillic) translations by @bzzrak in https://github.com/MicaForEveryone/MicaForEveryone/pull/289
* Translating Vietnam/vi language by @chefkozaki in https://github.com/MicaForEveryone/MicaForEveryone/pull/291
* fix(UI/Strings/zh-Hant): accurate based on Microsoft terminologies by @ritxsen in https://github.com/MicaForEveryone/MicaForEveryone/pull/293
* Add Azerbaijan translation by @thepoladov13 in https://github.com/MicaForEveryone/MicaForEveryone/pull/295

## New Contributors
* @MW1Z made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/226
* @Phyan made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/250
* @flandretw made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/261
* @seanardhana made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/264
* @baglayan made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/269
* @Zan1456 made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/270
* @lowl1f3 made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/282
* @GID0317 made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/283
* @jay900604 made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/284
* @bzzrak made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/289
* @chefkozaki made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/291
* @ritxsen made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/293
* @thepoladov13 made their first contribution in https://github.com/MicaForEveryone/MicaForEveryone/pull/295

**Full Changelog**: https://github.com/MicaForEveryone/MicaForEveryone/compare/v1.3.0.0...v1.3.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/MicaForEveryone/MicaForEveryone/releases/tag/v1.3.1.1'>v1.3.1.1</a>.</em>

**Comment by Skipper0X** [comment]
this release is still crashing when adding the new process rule or class rule

**Reply by Skipper0X** [reply]
can i see the logs anywhere, is there a log file locally ?

**Reply by xmine64** [reply]
It's a bug in MFE's UI, I'll fix it as soon as I can. (#307)

<img width="688" alt="image" src="https://github.com/MicaForEveryone/MicaForEveryone/assets/21140798/d5e5dca3-790d-4b93-a4bf-d2ec0def3225">

**Reply by dongle-the-gadget** [reply]
I'm not sure why you need to dynamically create an auto suggest box though.

**Reply by xmine64** [reply]
Honestly I don't remember either, but I had a reason

**Reply by Dhyfer1** [reply]
I came to say the same. i have the same problem as @Skipper0X and now i am using the old version. I hope that in the next version of MFE this bug will be corrected

**Comment by Dhyfer1** [comment]
Why is this version still using .net core 3.1? It is an old version and .net core 6.0, 7.0 and 8.0 are the latest versions

**Reply by xmine64** [reply]
See #209

**Reply by Dhyfer1** [reply]
I understand, it's okay. But anyway, it's about time for MFE to be updated with the latest version of .net core

**Reply by dongle-the-gadget** [reply]
It's not like we don't care. Unfortunately WinUI 3 isn't a viable choice for us right now.
